src/user/dashboard/navigation/navigation.js

```diff
@@ -1,12 +1,15 @@
 import React, { Component } from "react";
-import { ListGroup, Button } from "react-bootstrap";
-import { NavLink } from 'react-router-dom';
+import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import { DonutTitleSmall } from "../../../donutTitle/donutTitle";
 import "./navigation.scss";
 import {Logout} from "../../profile/popups/logout";
 import logo from "../../../svgs/logout.svg";
 import {Info} from "../../integrations/NameForm";
+import List from '@material-ui/core/List';
```

Comment:
Use desctructuring here

Reply:
Yeah sure there are some small changes I must do. I thought of doing it when I make the final commit. I’ll do it now anyway😁

Reply:
yeah cool!